src/utils/debug.cpp

```diff
@@ -215,15 +270,16 @@ bool onEvent(const SEvent &event)
 
             mnu->addItem(L"Adjust values", DEBUG_VISUAL_VALUES);
 
-            mnu->addItem(L"Profiler",DEBUG_PROFILER);
+            mnu->addItem(L"Profiler", DEBUG_PROFILER);
             if (UserConfigParams::m_profiler_enabled)
                 mnu->addItem(L"Toggle capture profiler report",
                              DEBUG_PROFILER_GENERATE_REPORT);
             mnu->addItem(L"Do not limit FPS", DEBUG_THROTTLE_FPS);
-            mnu->addItem(L"FPS",DEBUG_FPS);
+            mnu->addItem(L"Show FPS", DEBUG_FPS);
```

Comment:
Not fully accurate since it also hides, if you want to change the label then "Toggle" would be better